broker/binds/hussh.py

```diff
@@ -71,7 +71,7 @@ def __init__(self, **kwargs):
 
     @staticmethod
     def _set_destination(source, destination):
-        dest = destination or source
+        dest = str(destination or source)
         if dest.endswith("/"):
             dest = dest + Path(source).name
```

Comment:
You still keep `source` unconverted to string. 
it can be required for `Path()` to convert it to string.

```suggestion
            dest = dest + Path(str(source)).name
```